spec/rails_helper.rb

```diff
@@ -21,6 +22,9 @@
   config.allow_url 'cdn.ravenjs.com'
 end
 
+Capybara::Screenshot.webkit_options = { width: 1400, height: 768 }
+Capybara.save_and_open_page_path = "tmp/screenshots/"
```

Comment:
Just need to add `Capybara::Screenshot.prune_strategy = :keep_last_run`.

I only considered it running on travis, where it'll automatically destroy everything. But it could fill up locally after a while.